yt_dlp/extractor/nexx.py

```diff
@@ -77,6 +80,7 @@ class NexxIE(InfoExtractor):
             'timestamp': 1518614955,
             'upload_date': '20180214',
         },
+        'skip': 'Now works with arc API'
```

Comment:
why skip this?

Reply:
bc reason to add test is now untrue, but ig ill just remove that comment instead